test/fake-repos/fake-mixin-proto/fake_mixin.proto

```diff
@@ -0,0 +1,8 @@
+// This is a fake proto file for testing.
+
+syntax = "proto3";
+
+package google.fake_mixin
```

Comment:
I don't think so. It's really only the presence of this file that matters from the artman perspective. We test the mixing-in in toolkit.